### PR TITLE
[stable/jenkins] fix JAVA_OPTS when config auto-reload is enabled

### PR DESCRIPTION
**Pull request by kimxogus**
Signed-off-by: Taehyun Kim <kgyoo8232@gmail.com>

No

#### What this PR does / why we need it:
This pr fixes helm upgrade failure(helm 2.16.1).
```plain
UPGRADE FAILED
Error: YAML parse error on jenkins/templates/jenkins-master-deployment.yaml: error converting YAML to JSON: yaml: line 95: did not find expected key
Error: UPGRADE FAILED: YAML parse error on jenkins/templates/jenkins-master-deployment.yaml: error converting YAML to JSON: yaml: line 95: did not find expected key
```

#### Which issue this PR fixes
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
  - fixes #

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[stable/mychartname]`)

### CODE REVIEW

File: stable/jenkins/templates/jenkins-master-deployment.yaml [hunk 1]
```diff
@@ -168,7 +168,9 @@ spec:
             - name: JAVA_OPTS
               value: >
                 {{ default "" .Values.master.javaOpts }}
-                {{- if .Values.master.sidecars.configAutoReload.enabled }} -Dcasc.reload.token=$(POD_NAME) {{end}}
+                {{- if .Values.master.sidecars.configAutoReload.enabled }}
```
Comment (torstenwalter) [inline stable/jenkins/templates/jenkins-master-deployment.yaml]: @kimxogus What's the change except formatting here?

Comment (torstenwalter) [reply stable/jenkins/templates/jenkins-master-deployment.yaml]: If it's about having it on a new line then we could simply change it from `{{- if` to `{{ if`

Comment (kimxogus) [reply stable/jenkins/templates/jenkins-master-deployment.yaml]: Removed `-`.
There's no change except here

**Pull request merged**